website/content/en/preview/provisioner.md

```diff
@@ -59,6 +65,22 @@ spec:
   provider: {}
 ```
 
+## Node deprovisioning 
+
+If neither of these values are set, Karpenter will *not* delete instances. It is recommended to set the `ttlSecondsAfterEmpty` value, to enable scale down of the cluster. 
+
+### spec.ttlSecondsUntilExpired
+
+Setting a value here enables node expiry. After nodes reach the defined age in seconds, they will be deleted, even if in use. This enables nodes to effectively be periodically "upgraded" by replacing them with newly provisioned instances. This functionality may be called "repaving" or "rehydrating". 
```

Comment:
Can we delete the last sentence about `repaving` and `rehydrating` ?

Reply:
I'm keeping this in because the PM/DA team did get an explicit question about "repaving" from a customer.

Reply:
Agree with Brandon. This isn't a term of art, even if our customers call it one. We should avoid proliferating nonstandard terms.

Reply:
It's commonly used in the financial industry. The karpenter docs should, where appropriate, use specialized or niche terminology if it helps customers discover how karpenter can help them.